### PR TITLE
cephadm: update default monitoring container images

### PR DESCRIPTION
**Pull request by s0nea**
Prometheus server: ses/7/prometheus/prometheus-server:2.27.1
Node exporter: ses/7/prometheus/prometheus-node-exporter:1.1.2
Alertmanager: ses/7/prometheus/prometheus-alertmanager:0.21.0

Signed-off-by: Tatjana Dehler <tdehler@suse.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Review by p-se — APPROVED** [review]

**Comment by p-se** [comment]
@tserong ping

File: src/cephadm/cephadm [hunk 1]
```diff
@@ -51,9 +51,9 @@ FuncT = TypeVar('FuncT', bound=Callable)
 DEFAULT_IMAGE = 'quay.io/ceph/ceph:v16'
 DEFAULT_IMAGE_IS_MASTER = False
 DEFAULT_IMAGE_RELEASE = 'pacific'
-DEFAULT_PROMETHEUS_IMAGE = 'quay.io/prometheus/prometheus:v2.18.1'
-DEFAULT_NODE_EXPORTER_IMAGE = 'quay.io/prometheus/node-exporter:v0.18.1'
-DEFAULT_ALERT_MANAGER_IMAGE = 'quay.io/prometheus/alertmanager:v0.20.0'
+DEFAULT_PROMETHEUS_IMAGE = 'registry.suse.com/ses/7/prometheus/prometheus-server:2.27.1'
+DEFAULT_NODE_EXPORTER_IMAGE = 'registry.suse.com/ses/7/prometheus/prometheus-node-exporter:1.1.2'
+DEFAULT_ALERT_MANAGER_IMAGE = 'registry.suse.com/ses/7/prometheus/prometheus-alertmanager:0.21.0'
```
Comment (mgfritch) [inline src/cephadm/cephadm:56]:
```suggestion
DEFAULT_ALERT_MANAGER_IMAGE = 'registry.suse.com/ses/7.1/prometheus/prometheus-alertmanager:0.21.0'
```

wondering if these should be under a `7.1` path or similar ?

**Review by mgfritch — COMMENTED** [review]
I'm thinking we might want these

File: src/cephadm/cephadm (hunk 1 above)
Comment (s0nea) [reply src/cephadm/cephadm:56]: I took the `.../ses/7/...` path because that's where all other images lie. Or did I get anything wrong?

Comment (tserong) [reply src/cephadm/cephadm:56]: I think when this PR was originally opened, we hadn't decided on 7 vs 7.1, but we've since decided to use the 7.1 paths.

Comment (s0nea) [reply src/cephadm/cephadm:56]: Yes, you're right! I'll need to update it.

**Comment by tserong** [comment]
Obsoleted by #459

**Comment by s0nea** [comment]
> Obsoleted by #459

Thank you for closing! :slightly_smiling_face:

**Pull request closed without merging**